File: samples/server/petstore/java-msf4j/src/main/java/org/openapitools/api/impl/StoreApiServiceImpl.java
package org.openapitools.api.impl;

import org.openapitools.api.*;
import org.openapitools.model.*;

import java.util.Map;
import org.openapitools.model.Order;

import java.util.List;
import org.openapitools.api.NotFoundException;

import java.io.InputStream;

import org.wso2.msf4j.formparam.FormDataParam;
import org.wso2.msf4j.formparam.FileInfo;

import javax.ws.rs.core.Response;
import javax.ws.rs.core.SecurityContext;

@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaMSF4JServerCodegen")
public class StoreApiServiceImpl extends StoreApiService {
    @Override
    public Response deleteOrder(String orderId
 ) throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }
    @Override
    public Response getInventory() throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }
    @Override
    public Response getOrderById(Long orderId
 ) throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }
    @Override
    public Response placeOrder(Order body
 ) throws NotFoundException {
        // do some magic!
        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
    }
}

File: samples/server/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/api/StoreApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.Order
import org.springframework.stereotype.Service
@Service
class StoreApiServiceImpl : StoreApiService {

    override fun deleteOrder(orderId: String): Unit {
        TODO("Implement me")
    }

    override fun getInventory(): Map<String, Int> {
        TODO("Implement me")
    }

    override fun getOrderById(orderId: Long): Order {
        TODO("Implement me")
    }

    override fun placeOrder(body: Order): Order {
        TODO("Implement me")
    }
}
